src/game/etj_commands.cpp

```diff
@@ -2222,6 +2224,112 @@ bool RockTheVote(gentity_t *ent, Arguments argv) {
   return true;
 }
 
+bool addCustomVote(gentity_t *ent, Arguments argv) {
+  const int clientNum = ClientNum(ent);
+
+  const auto def = std::move(
+      ETJump::CommandParser::CommandDefinition::create(
+          "add-customvote", "Creates a new custom map vote list.")
+          .addOption("name", "Name of the custom vote list",
+                     ETJump::CommandParser::OptionDefinition::Type::Token, true)
+          .addOption("full-name", "Full name, displayed in the callvote text.",
+                     ETJump::CommandParser::OptionDefinition::Type::MultiToken,
+                     true)
+          .addOption("maps", "Maps to include in the list, space delimited.",
+                     ETJump::CommandParser::OptionDefinition::Type::MultiToken,
+                     true));
+
+  const auto optCommand =
+      ETJump::getOptCommand("add-customvote", clientNum, def, argv);
+
+  if (!optCommand.hasValue()) {
+    return false;
+  }
+
+  auto command = optCommand.value();
+
+  const std::string &name = command.options["name"].text;
```

Comment:
can these fail?

Reply:
No, `optCommand` has no value if name is empty so the if check above fails.